.github/workflows/vb_script.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 import argparse
 import pennylane as qml
-pl_version = '"' + qml.version() + '"\n'  # we expect PL v0.X.0 here if we're releasing v0.X.0 of the plugin 
+pl_version = '"' + qml.version() + '"'
```

Comment:
Maybe worth leaving a modified comment in?

`# we expect PL v0.X.0 here if we're releasing v0.X+1.0 of the plugin`

Reply:
```suggestion
pl_version = '"' + qml.version() + '"' # we expect PL v0.X.0 here if we're releasing v0.X+1.0 of the plugin
```